docs/development.md

```diff
@@ -179,6 +180,27 @@ To "log out", submit the following API call:
 POST http://localhost:8080/_ah/login?action=Log+Out
 ```
 
+## Setting up reCAPTCHA
+
+TEAMMATES uses reCAPTCHA on the `Recover Session Links` page (`/web/front/help/session-link-recovery`).
+By default, we use an [official test key](https://developers.google.com/recaptcha/docs/faq#id-like-to-run-automated-tests-with-recaptcha-what-should-i-do) for development purposes and the reCAPTCHA widget looks like the one shown in the figure below.
+ 
+![recaptcha-test-widget.png](images/recaptcha-test-widget.png)
+ 
+However, this should be changed for production environments:
+
+1. Go to the Google [reCAPTCHA website](https://www.google.com/recaptcha/intro/v3.html) and click `Admin Console`. You may be prompted to sign in with your Google account.
+1. Go to the `Register a new site` page by clicking the `+` icon, if you are not redirected automatically.
+1. Fill up the form and choose your [reCAPTCHA type](https://developers.google.com/recaptcha/docs/versions).
```

Comment:
This instruction is too detailed and is prone to being outdated. Even one line like "Register for CAPTCHA in [this website](https://www.google.com/recaptcha/admin). You may be prompted to sign in with your Google account." is good enough.

Everything else is CAPTCHA-specific implementation and such documentation does not belong in this repo.